MainModule/Client/UI/Colorize.rbxmx

```diff
@@ -99,86 +96,73 @@ return function(gui, guiData, gTable, env)
 		ScrollingFrame = true;
 	}
 	
+	local contents = {}
+	local OrgColors = {}
+	
 	local function getCont(obj)
-		for i,v in pairs(obj:GetChildren()) do 
-			if classes[v.ClassName] then
-				table.insert(contents,v)
+		for i,v in ipairs(obj:GetChildren()) do 
```

Comment:
We recently removed all pairs() and ipairs() functions from our code.

Pleas change `for i,v in ipairs(obj:GetChildren())` to `for i,v in obj:GetChildren()` and re-test your code.